RELEASENOTES.md

```diff
@@ -1,18 +1,40 @@
+
+This release also introduces two new amendments: `fixQualityUpperBound` and `RequireFullyCanonicalSig`.
+ 
+**New and Updated Features**
+- Native [gRPC API](https://grpc.io/) API support. Currently, this API provides a subset of the full `rippled` [API](https://xrpl.org/rippled-api.html). You can enable the gRPC API on your server with a new configuration stanza. 
+- API Versioning which allows for future breaking change of RPC methods to co-exists with existing versions.
+- Nodes now receive and broadcast UNLs over the peer network under various conditions. see `rippled` [#3072](https://github.com/ripple/rippled/pull/3072)
+- Augmented `submit` method to include additional details on the status of the command.
+- Improved `tx` method to allow the search to specify a range of ledgers using the `min_ledger` and `max_ledger` fields in the request.
+- new `validator_info` method which returns information pertaining to the current validators keys, manifest sequence, and domain.
+- new `manifest` method which looks up manifest information for the specified key (either master or ephemeral).
+- Introduce handshake protocol for compression negotiation (compression is not implemented at this point) and other minor improvements. 
+- Remove various old conditionals introduced by amendments.
+- Add `getRippledInfo` to packages to include info gathering script in `rippled` Linux packages.
+
+**Bug Fixes**
+- The `fixQualityUpperBound` amendment which fixes a bug in unused code for estimating the ratio of input to output of individual steps in cross-currency payments.
+- The `RequireFullyCanonicalSig` which changes the signature requirements for the XRP Ledger protocol so that non-fully-canonical signatures are no longer valid in any case. This protects against transaction malleability on all transactions, instead of just transactions with the tfFullyCanonicalSig flag enabled. Without this amendment, a transaction is malleable if it uses a secp256k1 signature and does not have tfFullyCanonicalSig enabled. Most signing utilities enable tfFullyCanonicalSig by default, but there are exceptions. With this amendment, no single-signed transactions are malleable. (Multi-signed transactions may still be malleable if signers provide more signatures than are necessary.) All transactions must use the fully canonical form of the signature, regardless of the tfFullyCanonicalSig flag. Signing utilities that do not create fully canonical signatures are not supported. All of Ripple's signing utilities have been providing fully-canonical signatures exclusively since at least 2014. For more information, see `rippled` issue [#3042](https://github.com/ripple/rippled/issues/3042).
```

Comment:
Perhaps append "amendment" to `RequireFullyCanonicalSig`.  Also remove "in any case" from the first sentence.  So the first sentence would read...

"The `RequireFullyCanonicalSig` amendment which changes the signature requirements for the XRP Ledger protocol so that non-fully-canonical signatures are no longer valid."

Reply:
Yes, updated. It reads better now!